pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -418,6 +418,58 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: PersistentVolumeControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// PersistentVolumeBinder.volumeController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("persistentvolumes"),
+				},
+				// PersistentVolumeBinder.syncVolume()
+				{
+					Verbs:     sets.NewString("get", "update", "create", "delete"),
+					Resources: sets.NewString("persistentvolumes"),
+				},
+				// PersistentVolumeBinder.syncVolume()
+				{
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("persistentvolumes/status"),
+				},
+				// PersistentVolumeBinder.claimController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("persistentvolumeclaims"),
+				},
+				// PersistentVolumeBinder.syncClaim()
+				{
+					Verbs:     sets.NewString("get", "update"),
+					Resources: sets.NewString("persistentvolumeclaims"),
+				},
+				// PersistentVolumeBinder.syncClaim()
+				{
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("persistentvolumeclaims/status"),
+				},
+				// PersistentVolumeRecycler.reclaimVolume() -> handleRecycle()
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("pods"),
+				},
+				// PersistentVolumeRecycler.reclaimVolume() -> handleRecycle()
+				{
+					Verbs:     sets.NewString("get", "create", "delete"),
```

Comment:
per the other comment, added "get" here, which follows where "get" lives in the other defined policies.